pallets/rc-migrator/src/conviction_voting.rs

```diff
@@ -250,3 +250,38 @@ pub mod alias {
 		ValueQuery,
 	>;
 }
+
+impl<T: Config> crate::types::RcMigrationCheck for ConvictionVotingMigrator<T> {
+	type RcPrePayload = Vec<RcConvictionVotingMessageOf<T>>;
+
+	fn pre_check() -> Self::RcPrePayload {
+		let mut messages = Vec::new();
+
+		// Collect VotingFor
+		for (account_id, class, voting) in alias::VotingFor::<T>::iter() {
+			messages.push(RcConvictionVotingMessage::VotingFor(account_id, class, voting));
```

Comment:
for conviction voting we filter zero votings, this test should fail on ah post check

Reply:
You're right! Fixed, thanks